charts/foundations/templates/klum.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if or (not .Values.foundations) (not .Values.foundations.disable) (not .Values.foundations.disable.klum) }}
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: GitRepository
+metadata:
+  name: klum
+spec:
+  interval: 30m
+  url: https://github.com/jadolg/klum
+  ref:
+    tag: v0.8.1
+  ignore: |
+    # exclude all
+    /*
+    # include deploy.yaml
+    !/deploy.yaml
+---
+# Makes sure nothing crashes while keeping all defaults empty and the code simple
+# {{- if not .Values.klum }}{{- set .Values "klum" (dict) }}{{ end }}
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: klum
+  namespace: foundations
+spec:
+  targetNamespace: foundations
+  serviceAccountName: flux-admin
+  interval: 1h
+  prune: true
```

Comment:
unsure, what happens if the upstream move the deploy.yaml?

Reply:
If they delete the deploy.yaml from the tagged version then we lose all users (slightly destructive for ops).

I don't really see an advantage to pruning this though you're right, it doesn't make anything *more secure*.